packages/web3-eth/src/index.js

```diff
@@ -36,6 +36,8 @@ var Iban = require('web3-eth-iban');
 var Accounts = require('web3-eth-accounts');
 var abi = require('web3-eth-abi');
 
+var util = require('util');
```

Comment:
Is this module available in the browser? As we compile to browser, we need to make sure its available there as well.

Reply:
Good question. I'm not familiar w/ how you're building for the browser yet, but since `util` is part of the node runtime, I'd expect most tools like `browserify` and `webpack` include shims for it. I'll double check and get back to you.

Reply:
It appears that you're using browserify, and it appears (per browserify/browserify#1503) that browserify will pull in the entire `util` package in this case.

If you'd rather not pull in the entire util module, we can do here what was suggested on that issue, use the `inherits` package instead of `util.inherits`. In the ES6 branch you can ditch that as well, and just use the `extends` keyword.

Reply:
It would be great if we can avoid using util here and use something natively existing in ES5 already